tools/setup_test.rs

```diff
@@ -63,35 +63,35 @@ fn dl_file(url: &str, to_file: &str, max_size: u64) -> Result<(), DlFileError> {
 
 fn main() -> Result<(), std::io::Error> {
 
-    println!("Using tag: {} for release of massa-unit-tests-src repo...", TAG);
+    // println!("Using tag: {} for release of massa-unit-tests-src repo...", TAG);
 
-    let path = format!("massa_unit_tests_{}.tar.gz", TAG);
-    let url = format!("https://github.com/massalabs/massa-unit-tests-src/releases/download/{}/{}", TAG, path);
+    // let path = format!("massa_unit_tests_{}.tar.gz", TAG);
+    // let url = format!("https://github.com/massalabs/massa-unit-tests-src/releases/download/{}/{}", TAG, path);
 
-    let extract_folder = format!("extract_massa_unit_tests_src_{}", TAG);
+    let extract_folder = format!("extract_massa_unit_tests_src_LOCAL");
 
     if Path::new(&extract_folder).exists() {
         println!("Please remove the folder: {} before runnning this script", extract_folder);
         std::process::exit(1);
     }
 
-    println!("Downloading: {}...", url);
-    let res = dl_file(&url, &path, ARCHIVE_MAX_SIZE);
+    // println!("Downloading: {}...", url);
+    // let res = dl_file(&url, &path, ARCHIVE_MAX_SIZE);
```

Comment:
The script should by default download from github. the 'local' option is only for developers for quick iteration so better to add an command line option to active it

Reply:
Same as above